app/javascript/src/case_emancipation.js

```diff
@@ -62,6 +62,43 @@ function saveCheckState (action, checkItemId) {
     })
 }
 
+export function manageTogglerText ($parent) {
+  const categoryCollapseIcon = $parent.find('.category-collapse-icon')
+
+  if ($parent.attr('data-is-open') === 'true') {
+    categoryCollapseIcon.text('–')
+  } else if ($parent.attr('data-is-open') === 'false') {
+    categoryCollapseIcon.text('+')
+  }
+}
+
+export function openChildren ($parent) {
+  const categoryOptionsContainer = $parent.siblings('.category-options')
+
+  categoryOptionsContainer.show()
+  $parent.attr('data-is-open', 'true')
+}
+
+export function closeChildren ($parent) {
+  const categoryOptionsContainer = $parent.siblings('.category-options')
```

Comment:
**nitpick**: the functions point that they receive a jquery element by using `$` at the start of the variable name (`$parents`). Then we store a different jquery element without using `$` (`categoryOptionsContainer`). I think being consistent would make code clearer (so either using `$` for all jquery elements (`$parents` and `$categoryOptionsContainer`, or using nothing)